pages/common/qpdf.md

```diff
@@ -23,6 +23,6 @@
 
 `qpdf --rotate={{90:2,4,6}} --rotate={{180:7-8}} {{input.pdf}} {{output.pdf}}`
 
-- Remove the password from a password protected file:
+- Remove the password from a password-protected file:
```

Comment:
I dislike this change.

Reply:
This one is up for debate that we'll see what others say.

I can vouch in the Oxford Dictionary it's "password-protected".

https://www.lexico.com/definition/password-protected

Reply:
I prefer `password-protected`. A hypen can often improve readability.

Reply:
Generally, I think the hyphen is better because the word "password" is being used as a modifier.
In this case, it's "a protected file", but we modify the word protected to say how it is protected, hence "password-protected".

Similarly to "human-readable", the word human is used as a modifier of the word "readable".

More on the topic:
https://en.wikipedia.org/wiki/English_compound#Hyphenated_compound_modifiers

Even on the Wikipedia page is does note that different style guides have different recommendations for it. The only real rule is, use a hyphen if the text might be ambiguous without it.